.haml-lint.yml

```diff
@@ -1,3 +1,24 @@
 linters:
   LineLength:
     max: 120
+
+  InstanceVariables:
+    enabled: true
+    file_types: partials
+    matchers:
+      all: .*
+      partials: \A_.*\.haml\z
+
+  SelectorDepth:
+    max_depth: 3
+
+  IdNames:
+    enabled: true
+    style: lisp_case
+
+  rules:
+  class-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+      ignore:
+        - class_name_b
```

Comment:
i've not seen this syntax before -- could a comment help make it clearer for the next person who works here?